specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -1572,6 +1572,16 @@ export const userPreferenceDefinitions = {
               },
             ],
           }),
+          exportCSVutf8BOM: definePref<boolean>({
```

Comment:
try not to use consecutive capitalized letters as they create inconsistency and readability problems in the long run

```suggestion
          exportCsvUtf8Bom: definePref<boolean>({
```